frontend/ui/lib/widgets/trip_list_widget.dart

```diff
@@ -27,9 +27,18 @@ class TripListWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ListView(
-      children: trips.map(toListTile).toList(),
-    );
+    return Column(children: [
+      RaisedButton(
```

Comment:
Can we not put this on the TripListWidget and instead put it on the page (or something that wraps the List?)

Reply:
Yeah I can make this a seperate widget.